lib/hooks/queries/useInfiniteMarkets.ts

```diff
@@ -81,6 +78,7 @@ export const useInfiniteMarkets = (
             status_in: statuses.length === 0 ? undefined : statuses,
             tags_containsAny: tags?.length === 0 ? undefined : tags,
             baseAsset_in: currencies?.length !== 0 ? currencies : undefined,
+            scoringRule_not_eq: ScoringRule.Parimutuel,
```

Comment:
I think it would be best to add this to `marketMetaFilter` filter so it covers other parts of the app

Reply:
Should be fixed now